arm-sql/2015-05-01/swagger/sql.json

```diff
@@ -0,0 +1,227 @@
+        },
+        "disabledAlerts": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The disable alerts of the Azure SQL Security alert policy."
+        },
+        "emailAddresses": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The list of email addresses to send the alert."
+        },
+        "emailAccountAdmins": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Value that indicated whether to email the azure account admins."
+        }
+      },
+      "description": "Represents the properties of an Azure SQL Security alert policy."
+    },
+    "ServerSecurityAlertPolicyCreateOrUpdateParameters": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ServerSecurityAlertPolicyProperties",
+          "description": "Properties of the request."
+        }
+      },
+      "required": [
+        "properties"
+      ],
+      "description": "Create or update server security alert policy parameters."
+    },
+    "ServerSecurityAlertPolicy": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ServerSecurityAlertPolicyProperties",
+          "description": "Represents the properties of the resource."
+        }
+      },
+      "description": "Represents an Azure SQL Server Security Alert Policy."
+    },
+    "Resource": {
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource Id"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name"
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type"
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "x-ms-azure-resource": true
+    },
+    "SubResource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource Id"
+        }
+      },
+      "x-ms-azure-resource": true
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The subscription credentials which uniquely identify Microsoft Azure subscription."
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Client Api Version."
+    },
+    "ResourceGroupNameParameter": {
```

Comment:
All the parameters defined in the global section will become properties of the generated client. You would not want `resourceGroupName` and `ServerNameParameter` to be properties of the client. Please refer to the documentation over here: https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md#globalParam.

In future we plan to add an extension called `x-ms-client-property : true/false` which will indicate the generator to do the right thing.